src/utilities/price.test.js

```diff
@@ -10,7 +10,7 @@ describe('Price Formatting', () => {
         expect(convertPenniesToDollars(999)).toEqual("9.99");
         expect(convertPenniesToDollars(1000)).toEqual("10.00");
         expect(convertPenniesToDollars(10000)).toEqual("100.00");
-        expect(convertPenniesToDollars(100000)).toEqual("1000.00");
+        expect(convertPenniesToDollars(100000)).toEqual("1,000.00");
```

Comment:
Can we add a few more tests here to make sure it's working as expected?

Add these cases please: 
10,000
100,000
1,000,000

Reply:
Good job otherwise!!